### DISCUSSION TITLE
Pulling OpenVAS data to External sources

### DISCUSSION THREAD

**Discussion by krgroupinc** [Secure-Compliance-Solutions-LLC/GVM-Docker#338] [Q&A]
Good morning,

I'm inquiring on several topics, mainly getting Scan results data out of GVM. We're not wanting to use email, as more overhead with 10-12 GVMs scattered around. I thought about a SIEM like Graylog, but having issues connecting to Postgres Database.. I've attempted changing the DB Password, but still seeing a failure, perhaps I'm using the wrong username.

I'm curious how you're pulling data out of GVM, we ultimately want to pull a PDF every 30 days for archieval purposes.

Thanks.

**Comment by Dexus** [comment]
Maybe you should have a view on the Logstash repository if you like to export those data, or you use the python-gvm module and connect to gvm to export what you need.

**Reply by krgroupinc** [reply]
Dexus,

Thanks for the information. I was aiming to use Logstash, but the Elastic Search function gave me the impression I"m needing sn ES instance setup. Is that not accurate?

EDIT:

I feel silly.. I have the ES  instance running, however, am encountering an issue with a certificate verification between ELastic Search and Logstash.

**Reply by Dexus** [reply]
I'm not using it, so currently I'm not able to support you a more aside of this hint.
